Rating.js

```diff
@@ -1,88 +1,99 @@
 import React from 'react';
-import {ToggleStarBorder, ToggleStar} from 'material-ui/svg-icons';
-import {colors} from 'material-ui/styles';
+import { StarBorder, Star } from 'material-ui-icons';
+import colors from 'material-ui/colors';
+import { orange, grey } from 'material-ui/colors';
 
 const styles = {
-    editable: {
-        cursor: 'pointer'
-    }
+  editable: {
+    cursor: 'pointer',
+  },
 };
 
-const defaultValues = [1,2,3,4,5];
+const defaultValues = [1, 2, 3, 4, 5];
 
-const Ch = (props) => {
-    let {checked, hovered, readOnly=false, ...p} = props;
-    const st = readOnly ? {} : styles.editable;
-    if (checked)
-        return <ToggleStar style={st} color={colors.orange500} {...p}/>;
-    else if (hovered)
-        return <ToggleStarBorder style={st} color={colors.orange500} {...p}/>;
-    else
-        return <ToggleStarBorder style={st} color={colors.grey300} {...p}/>;
+const Ch = props => {
+  let { checked, hovered, readOnly = false, ...p } = props;
+  const st = readOnly ? {} : styles.editable;
+  if (checked) return <Star style={st} color={orange['500']} {...p} />;
+  else if (hovered)
+    return <StarBorder style={st} color={orange['500']} {...p} />;
+  else return <StarBorder style={st} color={grey['500']} {...p} />;
 };
 
 class Rating extends React.Component {
+  static defaultProps = {
+    onChange: () => console.log('hi'),
```

Comment:
It`ll be better to write some debug information for developer or some instructions m.b.
 instead of just "hi" ;)

Reply:
Yeah haha 😀 forgot I put that in there to be honest. I'll submit another PR to fix it. Also would love to get this package on NPM if I do the leg work of setting up a package.json for you will you publish it?

Reply:
Niiice. Do it and I`ll check, do merge and will publish

Reply:
Awesome! 🎉